sandy/fy.py

```diff
@@ -213,10 +213,9 @@ def _expand_zam(self):
                            dtype=int)
         return self.data.assign(Z=zam.Z, A=zam.A, M=zam.M)
 
-    def get_chain_fission_yields(self, zam, e):
+    def get_mass_chain(self, zam, e):
```

Comment:
there are two type of yields: `chain_yields` and `mass_yields`. not `mass_chain`